### PR TITLE
feat(Workspaces): hide notebooks menu entry for user with role VIEWER

### PR DESCRIPTION
**Pull request by cheikhgwane**
Hide JupyterHub menu entry on the sidebar for user with role VIEWER.

## Changes

Please list / describe the changes in the codebase for the reviewer(s).

- Hide menu entry
- Return not found when user try to access to /notebooks

## How/what to test

Create a Workspace Member with Role Viewer, connect with it and check that the JupyterHub menu entry is hidden and  try to access to /notebooks.

### CODE REVIEW

File: src/pages/workspaces/[workspaceSlug]/notebooks.tsx [hunk 1]
```diff
@@ -66,7 +66,7 @@ export const getServerSideProps = createGetServerSideProps({
     });
     await WorkspaceLayout.prefetch(client);
 
-    if (!data.workspace) {
+    if (!data.workspace || !data.workspace.permissions.manageMembers) {
```
Comment (qgerome) [inline src/pages/workspaces/[workspaceSlug]/notebooks.tsx:69]: Not sure if the perm is included in the query.

Comment (cheikhgwane) [reply src/pages/workspaces/[workspaceSlug]/notebooks.tsx]: Yes and the manageMembers was too restrictive. I've changed to all users with update permission

**Review by qgerome — APPROVED** [review]

**Review by qgerome — COMMENTED** [review]
Can you add a small test where the user has the permission to see the menu entry ?

**Review by qgerome — CHANGES_REQUESTED** [review]
Just to add the test

**Comment by cheikhgwane** [comment]
> Just to add the test

Yes, I'm on it. Currently facing some issues with the  apollo client cache

**Review by qgerome — APPROVED** [review]

**Pull request merged**